qcengine/programs/gamess/harvester.py

```diff
@@ -97,6 +103,19 @@ def harvest_outfile_pass(outtext):
             qcvar['MP2 CORRELATION ENERGY'] = mobj.group(3)
             qcvar['MP2 TOTAL ENERGY'] = mobj.group(4)
 
+        # Process MP2
+        mobj = re.search(
+            r'^\s+' + r'RESULTS OF 2ND-ORDER ZAPT CORRECTION' + r'\s*' +
+            r'^\s+' + r'E\(HF\)   ' + r'=\s+' + NUMBER + r'\s*' +
+            r'^\s+' + r'E\(ZAPT\) ' + r'=\s+' + NUMBER + r'\s*' +
+            r'^\s+' + r'-----------------------------------' + r'\s*' +
```

Comment:
note that you can do something like https://github.com/MolSSI/QCEngine/blob/master/qcengine/programs/cfour/harvester.py#L290 for dashed lines. it's more robust if the program decides to shorten/lengthen.